docs/installation/instructions.rst

```diff
@@ -1,8 +1,28 @@
 Installation Instructions
 =========================
 
-Ansible
--------
+Pulp 3 Ansible Galaxy Installation
+----------------------------------
+
+1. In order to be able to run playbook, ensure python3, ansible( >2.2) and libselinux-python are installed:
```

Comment:
libselinux-python is required to be installed on the target machine not the control machine:
http://docs.ansible.com/ansible/latest/intro_installation.html#managed-node-requirements